src/themes/teams/components/Radio/radioStyles.ts

```diff
@@ -0,0 +1,36 @@
+import { IComponentPartStylesInput, ICSSInJSStyle } from '../../../../../types/theme'
+
+const radioStyles: IComponentPartStylesInput = {
+  root: ({ props, variables }): ICSSInJSStyle => {
+    return {
+      display: 'inline-flex',
+      position: 'relative',
+      alignItems: 'center',
+      justifyContent: 'flex-end',
+      outline: 0,
+      minHeight: '2.5rem',
+    }
+  },
+
+  radio: ({ props, variables }): ICSSInJSStyle => {
+    return {
+      display: 'inline-block',
+    }
+  },
+
+  label: ({ props, variables }): ICSSInJSStyle => {
+    return {
+      alignItems: 'center',
```

Comment:
this one is unnecessary (has no effect for this `display` value)

Reply:
true, thanks, will fix